### DISCUSSION TITLE
How do I set a static title bar?

### DISCUSSION THREAD

**Discussion by ylbeethoven** [kovidgoyal/kitty#5670] [Q&A] [answered]
Hi,

I am having issue setting a static title bar which was working in previous version.

For example, I use below on my `.zshrc` file, this sets the title to Hello world automatically.

`echo -ne "\e]0;Hello World😺\a"`

Now if I type anything or go to any folder, kitty will display the directory path on title bar, as shown in below screenshot.
![Screenshot from 2022-11-15 18-14-25](https://user-images.githubusercontent.com/1949110/201853798-3e801a0b-6f72-4942-ae3c-8c6bfadcf3a0.png)

I don't have this issue before and it also works on Gnome Terminal
![Screenshot from 2022-11-15 18-16-14](https://user-images.githubusercontent.com/1949110/201853984-3f006b5c-9bea-427c-ad10-5b75ea7a97fd.png)

Is there a method to disable showing current directory path?

Thank you.

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/shell-integration/#configuration

**Reply by ylbeethoven** [reply]
Thanks.

I was searching options on the default `kitty.conf` and could not find the setting.

Anyway, for this particular use case, I need to set below on `kitty.conf`.

```
shell_integration no-title
```

**Reply by Pukimaa** [reply]
Hey! This doesn't seem to do the trick for me. I'm rocking Gnome 47 together with Wayland :)

**Reply by tullo-x86** [reply]
On MacOS, this setting _appears_ to work until I `ssh` into another machine, and then the SSH session sets the window title _and then doesn't change back after exiting the SSH session_... is this a bug or have I misunderstood the config?
